### PR TITLE
Change logic of L4 Strong Session Affinity

### PR DESCRIPTION
**Pull request by code-elinka**
Session Affinity Tracking Policy shouldn't
be updated if the flag for it wasn't set.

### CODE REVIEW

File: pkg/loadbalancers/l4netlb.go [hunk 1]
```diff
@@ -266,12 +266,13 @@ func (l4netlb *L4NetLB) provideIPv4HealthChecks(nodeNames []string, result *L4Ne
 // connectionTrackingPolicy returns BackendServiceConnectionTrafficPolicy
 // based on StrongSessionAffinity and IdleTimeoutSec
 func (l4netlb *L4NetLB) connectionTrackingPolicy() (*composite.BackendServiceConnectionTrackingPolicy, error) {
+	if !l4netlb.enableStrongSessionAffinity || !annotations.HasStrongSessionAffinityAnnotation(l4netlb.Service) {
+		return nil, nil
+	}
 	connectionTrackingPolicy := composite.BackendServiceConnectionTrackingPolicy{}
 	connectionTrackingPolicy.EnableStrongAffinity = annotations.HasStrongSessionAffinityAnnotation(l4netlb.Service)
```
Comment (mmamczur) [inline pkg/loadbalancers/l4netlb.go:273]: this now can be `connectionTrackingPolicy.EnableStrongAffinity = true` since false will trigger return above

Comment (code-elinka) [reply pkg/loadbalancers/l4netlb.go:273]: done, usually I prefer explicity :newspaper_roll:

**Pull request closed without merging**